### PR TITLE
To Do list: add & remove

### PR DESCRIPTION
**Pull request by PeteUgwu**
## To Do list: add & remove

- [ ] Create a new JavaScript file **todo.js** for the new functionality.
- [ ] Implement a function for adding tasks
- [ ] Implement a function for deleting tasks
- [ ] update READme file
- [ ] fix linters

### CODE REVIEW

File: src/todo.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    localStorage.setItem("todo", JSON.stringify(toDos));
+    this.loadTodo(toDos);
+  };
+
+  static loadTodo = (item) => {
+    let display = " ";
+    item.forEach((elem, i) => {
+      display += `
+      <div class="todo-check flex">
+      <div class="checkbox">
```
Comment (ichala) [inline src/todo.js]: - [ ] as mentioned by the requirements, kindly add the functionality of editing task descriptions, the adding and removing are working great only the updating function is missing 🙏

File: webpack.config.js [hunk 2]
```diff
@@ -6,11 +7,11 @@ module.exports = {
...
-  output: {
-    filename: "main.js",
-    path: path.resolve(__dirname, "dist"),
-    clean: true,
-  },
+  // output: {
+  //   filename: "main.js",
+  //   path: path.resolve(__dirname, "dist"),
+  //   clean: true,
+  // },
```
Comment (ichala) [inline webpack.config.js]: - [ ] kindly remove any commented code in your project to keep the code clean and professional 🙏

**Review by ichala — CHANGES_REQUESTED** [review]
Hi @PeteUgwu  
Good job so far! 
👏👏👏 
There are some issues that you still need to work on to go to the next project but you're almost there!

the-simpsons-homer-simpson

## To Highlight 🎉

- Website Deployed✔️ 

 ## Required Changes ♻️

- [ ] 🛑 **_Check the comments below_** 

## Optional suggestions

- N/A

**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ichala — APPROVED** [review]
Hi @PeteUgwu  ,
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  
 
68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966

# To Highlight 💪🏻

-  Changes Implemented ✔️ 



 Good job implementing the requirements  :+1: Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Optional Suggestions  :

- N/A


**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag  @ichala in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by 187jjay187 — COMMENTED** [review]
Great Code!!

**Pull request merged**